### PR TITLE
Fix for proper proc_path handling

### PR DESCRIPTION
**Pull request by obourdon**

### CODE REVIEW

File: mem/mem.go [hunk 1]
```diff
@@ -72,6 +79,9 @@ func Meta() *plugin.PluginMeta {
 // GetMetricTypes returns list of available metric types
 // It returns error in case retrieval was not successful
 func (mp *memPlugin) GetMetricTypes(cfg plugin.ConfigType) ([]plugin.MetricType, error) {
+	if procPath, ok := cfg.Table()["proc_path"]; ok {
```
Comment (marcin-krolik) [inline mem/mem.go]: For the sake of consistency consider using `config.GetConfigItem(cfg, "proc_path")`

**Comment by marcin-krolik** [comment]
LGTM

File: examples/tasks/mem-file.json [hunk 2]
```diff
@@ -12,7 +12,11 @@
                 "/intel/procfs/meminfo/mem_total": {},
                 "/intel/procfs/meminfo/mem_used": {}
             },
-            "config": {},
+            "config": {
+	      "/intel/procfs": {
+                "proc_path": "/proc"
```
Comment (elemoine) [inline examples/tasks/mem-file.json]: Don't you want to use `"/hostproc"` in the example?

**Comment by elemoine** [comment]
I've added a minor comment, otherwise looks good to me. Also, do you want to squash your two commits into one? I am not sure what the Snap policy is w.r.t. that.

File: examples/tasks/mem-file.json (hunk 2 above)
Comment (obourdon) [inline examples/tasks/mem-file.json]: I think it is best to keep example in a working state. This way, people can use the task right out of the box without any modifications. So /proc is the default which should be working and people can use any other customized value if need for use in docker for instance

Comment (elemoine) [inline examples/tasks/mem-file.json]: Oh I see. That makes sense.

**Comment by obourdon** [comment]
@intelsdi-x/snap-maintainers could you please review and merge, thanks

**Pull request merged**